docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+
+  postgres:
+    image: postgres
+    container_name: pgsql-dev
+    networks:
+      - redarc
+    environment:
+      POSTGRES_PASSWORD: test1234
+    volumes:
+      - pgredarc01:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  redarc:
+    build:
+      context: https://github.com/Yakabuff/redarc.git
```

Comment:
I had to change context to . to get the container to run properly but it should be fine once I merge it right?

Reply:
The context being changed to the git should have the docker-compose clone the current repo to be used for the build by default. That way only the docker-compose.yml is required to get setup and it will take care of the clone automatically. This should work even with the current commit. I was able to download the docker-compose.yml from my repo with no changes and it was able to pull the latest and build correctly:

[+] Building 66.8s (19/19) FINISHED
 => [internal] load git source https://github.com/Yakabuff/redarc.git                                              1.7s
 => [internal] load metadata for docker.io/library/ubuntu:22.04                                                    0.8s
 => CACHED [ 1/16] FROM docker.io/library/ubuntu:22.04@sha256:6120be6a2b7ce665d0cbddc3ce6eae60fe94637c6a66985312d  0.0s
 => [ 2/16] RUN apt-get update && apt-get install -y                                                               8.9s
 
So without any changes in a folder with just the docker-compose.yml I was able to get it to work, but if there's something weird or small I missed causing issues on yours let me know.

Reply:
But in the output, it's pulling the old Dockerfile with ubuntu as the base image instead of alpine?